velox/type/Conversions.h

```diff
@@ -418,6 +455,51 @@ struct Converter<TypeKind::VARCHAR, void, TRUNCATE, LEGACY_CAST> {
   static std::string cast(const bool& val) {
     return val ? "true" : "false";
   }
+
+  static void normalizeStandardNotation(std::string& str) {
+    if (!FLAGS_experimental_enable_legacy_cast &&
+        str.find(".") == std::string::npos && isdigit(str[str.length() - 1])) {
+      str += ".0";
+    }
+  }
+
+  static void normalizeScientificNotation(std::string& str) {
```

Comment:
It would be nice to document this method.

Reply:
Done